src/styles/main.scss

```
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import url(./blocks/calendar.scss);
```

```suggestion
@import "/blocks/calendar";
```

Don't need to use URL function in scss